src/test/java/uk/gov/hmcts/reform/hmc/service/LinkedHearingGroupServiceTest.java

```diff
@@ -377,4 +384,62 @@ private LinkedGroupDetailsAudit createGroupDetailsAuditEntity(long hearingGroupI
         }
 
     }
+
+    @Nested
+    @DisplayName("getHearingGroup")
+    class GetHearingGroup {
+        public static final String INVALID_REQUEST_NAME = "Invalid Name";
+        public static final String VALID_REQUEST_ID = "Request Name";
+
+        @Test
+        void shouldGetLinkedHearingGroupDetails() {
+            LinkedGroupDetails linkedGroupDetails =
+                generateLinkedGroupDetails(HEARING_GROUP_ID);
+            HearingEntity hearing1 = generateHearingEntity(HEARING_ID1,linkedGroupDetails);
+            HearingEntity hearing2 = generateHearingEntity(HEARING_ID2,linkedGroupDetails);
+
+            given(linkedGroupDetailsRepository.getLinkedGroupDetailsByRequestId(VALID_REQUEST_ID))
+                .willReturn(linkedGroupDetails);
+            given(hearingRepository.findByLinkedGroupId(HEARING_GROUP_ID))
+                .willReturn(List.of(hearing1, hearing2));
+
+            LinkedHearingGroupResponses linkedHearingGroupResponses =
+                service.getLinkedHearingGroupDetails(VALID_REQUEST_ID);
+            GroupDetails returnedGroupDetails = linkedHearingGroupResponses.getGroupDetails();
+            assertFalse(
+                returnedGroupDetails != null
+                    && returnedGroupDetails.getGroupName().isEmpty()
+                    && returnedGroupDetails.getGroupComments().isEmpty());
+        }
+
+        @Test
+        void shouldReturn404ErrorWhenRequestIdIsNotFound() {
```

Comment:
```suggestion
        void shouldReturnErrorWhenRequestIdIsNotFound() {
```
Looks like it's a 400 error, not 404 - but in the context of this test it's only checking the exception anyway so could just remove the code from the test name.